moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -170,9 +141,19 @@ void JogCalcs::startMainLoop(JogArmShared& shared_variables)
     shared_variables.tf_moveit_to_cmd_frame = tf_moveit_to_cmd_frame_;
     shared_variables.unlock();
 
-    // If paused, just keep the low-pass filters up to date with current joints so a jump doesn't occur when
-    // restarting
-    if (halt_outgoing_jog_cmds_)
+    //  Check for an initial jog command
+    if (wait_for_jog_commands)
+    {
+      shared_variables.lock();
+      // Check if there are any commands with valid timestamp
+      wait_for_jog_commands = shared_variables.command_deltas.header.stamp == ros::Time(0.) &&
```

Comment:
Is it possible to miss this initial jog command?

Reply:
I'm not sure I understand what you mean by this. Either a command has been received before this check or not. If not, we didn't really miss it because it was received after this check. In that case nothing happens and it will be picked up in the next loop cycle. Theoretically you would only miss commands if you publish them in a much higher rate than the jog control rate, but this seems unrelated to your question.

Reply:
Cool, just wanted to make sure you considered this